internal/support/notifications/metrics_system.go

```diff
@@ -0,0 +1,39 @@
+package notifications
+
+import (
+	"net/http"
+	"fmt"
+	"runtime"
+)
+
+func metricsHandler(w http.ResponseWriter, r *http.Request) {
+
+	if r.Body != nil {
+		defer r.Body.Close()
+	}
+
+	// The micro-service is to be considered the System Of Record (SOR) in terms of accurate information.
+	// Fetch metrics for the notifications service.
+	var m Monitor
+	var rtm runtime.MemStats
+
+	// Read full memory stats
+	runtime.ReadMemStats(&rtm)
+
+	// Miscellaneous memory stats
+	m.Alloc = rtm.Alloc
+	m.TotalAlloc = rtm.TotalAlloc
+	m.Sys = rtm.Sys
+	m.Mallocs = rtm.Mallocs
+	m.Frees = rtm.Frees
+
+	// Live objects = Mallocs - Frees
+	m.LiveObjects = m.Mallocs - m.Frees
```

Comment:
This should be calculated internally by the "Monitor" type, not by every service that performs the metrics operation.